src/lwc/utilTemplateBuilder/utilTemplateBuilder.js

```diff
@@ -53,10 +54,16 @@ import ACCOUNT_INFO from '@salesforce/schema/Account';
 import commonError from '@salesforce/label/c.commonError';
 import commonUnknownError from '@salesforce/label/c.commonUnknownError';
 
+import getDataImportSettings from '@salesforce/apex/UTIL_CustomSettingsFacade.getDataImportSettings';
+import getGiftEntryFeatureGateSettings from
```

Comment:
UTIL_CustomSettingsFacade.getGiftEntryFeatureGateSettings was included in previous commits, but I think is missing currently from your build